src/pg/sql/helper.js

```diff
@@ -0,0 +1,47 @@
+import sql, { raw, join } from 'sql-template-tag';
```

Comment:
(minor) I try to avoid names like `helpers`, `utils` etc. This file seems to be about generating small clauses of SQL, I suggest calling it `clauses.js`?